migrations/lib/importer/step.rb

```diff
@@ -82,6 +85,10 @@ def execute
 
     private
 
+    # Override in subclasses if necessary
+    def setup
+    end
```

Comment:
We more or less can achieve the same with `execute`, but I thought it would be nice to have a way of initializing variables without having to override the `initialize` method. It's a PITA to update method signatures in all child classes when we change the method in the `Step` class.